src/dappsExplorer/NoResults.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { DappExplorerEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Touchable from 'src/components/Touchable'
+import { DappFilter } from 'src/dapps/types'
+import Colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  filter: DappFilter | null
+  removeFilter: () => void
+  testID: string
+}
+
+export function NoResults({ filter, removeFilter, testID }: Props) {
+  const { t } = useTranslation()
+
+  const removePress = () => {
+    ValoraAnalytics.track(DappExplorerEvents.dapp_filter_remove)
+    removeFilter()
+  }
+
+  return (
+    <View testID={`${testID}/NoResults`} style={styles.favorites}>
+      <Text style={styles.filterAppliedText}>
+        <Trans
+          i18nKey={'dappsScreen.emptyResults.message'}
+          tOptions={{ filter: filter?.name.toUpperCase() }}
+        >
+          <Text style={styles.filterText} />
+        </Trans>
+      </Text>
+      <Touchable
+        onPress={removePress}
+        style={styles.removeFilterTouchable}
+        testID={`${testID}/NoResults/RemoveFilter`}
+      >
+        <Text style={styles.removeFilterText}>{t('dappsScreen.emptyResults.removeFilter')}</Text>
+      </Touchable>
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
+  favorites: {
+    flex: 1,
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    marginTop: Spacing.Thick24,
+  },
+  filterText: {
+    ...fontStyles.regular500,
+    fontSize: 12,
+    lineHeight: 20,
+    fontWeight: '500',
+    color: Colors.onboardingBlue,
+  },
+  filterAppliedText: {
+    ...fontStyles.regular500,
+    fontWeight: '400',
+    fontSize: 12,
+    lineHeight: 20,
+    color: Colors.dark,
+    paddingHorizontal: 4,
+  },
+  removeFilterTouchable: {
+    paddingVertical: 0,
+    paddingHorizontal: 8,
+    backgroundColor: Colors.gray2,
+    borderRadius: 16,
+  },
+  removeFilterText: {
+    ...fontStyles.regular500,
+    fontWeight: '500',
+    fontSize: 12,
+    lineHeight: 20,
```

Comment:
i think i commented on the previous PR but what do you think about just using the closest font that we have defined (e.g. small500). it means that when we update the design system, places like this will not be missed (since it has a bunch of custom styles that overwrite the design system). fwiw i think that not customising the design system styles keeps the integrity of the system + it's less code to maintain :) even if this means that we're off by a few pixels from the designs

Reply:
Fixed in e5341bcc2662f046e048379299861b43e3205765!